code/modules/admin/admin_verbs.dm

```diff
@@ -1186,3 +1187,15 @@ GLOBAL_PROTECT(admin_verbs_poll)
 		QDEL_NULL(segment.ai_controller)
 		segment.AddComponent(/datum/component/mob_chain, front = previous)
 		previous = segment
+
+/client/proc/spawn_as_mmi(mob/living/carbon/human/target in GLOB.human_list)
+	set category = "Debug"
+	set name = "Turn target into MMI"
+	set desc = "Turns something into an MMI, must be used on humans"
+	if(!check_rights(R_DEBUG))
+		return
+
+	var/obj/item/mmi/new_mmi = new(target.loc)
+	var/obj/item/organ/internal/brain/B = target.get_organ_slot(ORGAN_SLOT_BRAIN)
+	B.Remove(target)
+	new_mmi.attackby(B,target)
```

Comment:
new single letter variables are frowned upon, please replace `B` with something like `target_brain` or the like.

also, does this not delete the human after we MMI them? i feel like we should do this.

Reply:
I left it in because I actuely wouldf need the human body to test mechs, but I can remove it if desired

Reply:
it's pretty easy to ctrl+click your ghost to spawn in a new human if you adminghost out of the MMI right after doing this to make a new human body so i would rather you do delete the old human

Reply:
Want me to make it strip the targets stuff? or just delete em?